### ISSUE TITLE
[1.15.2] Mob Health Config Broken

### ISSUE THREAD

**Issue by Peach774** [McJty/TheOneProbe#389]
The config to disable mob health doesn't work. This is true for all versions of the mod for 1.15.2

**Comment by McJty** [comment]
Describe exactly what you're doing

**Comment by Peach774** [comment]
I set the config to "NOT"

**Comment by McJty** [comment]
MORE details please. Where do you set that?

**Comment by Peach774** [comment]
in theoneprobe-client config file, in the config folder, there are two lines 
"#Show mob health (0 = not, 1 = always, 2 = sneak)
showMobHealth = "NOT""
This is what I have it set to

**Comment by credomane** [comment]
The valid values are 0, 1, 2
If you want it to not show you set it to "0" not "not". The words are just the meaning of the values.

**Comment by Peach774** [comment]
> 
> 
> The valid values are 0, 1, 2
> If you want it to not show you set it to "0" not "not". The words are just the meaning of the values.

Yeah that straight up just doesn't work. If I set it to 0 instead of NOT, the config replaces the value with "NORMAL"

**Comment by credomane** [comment]
AH....I see what's up. The config system got rewritten semi-recently and appears to function a little differently than before. I was looking at the 1.12 source and the values I said are definitely the only allowed values. Now it seems that with the new config system that isn't the case anymore even though it says it is still the case.

I'm digging through the 1.15 source now. Either the docs need updated for the new accepted values or the new config system has some kinks in it. Give me a bit to see what it is actually expecting now...
